Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1583,6 +1645,572 @@ def is_ip_valid(s, accept_v6_ips=False):
         return True
 
 
+class Indicator(object):
+    """
+    interface class
+    """
+    @abstractmethod
+    def to_context(self):
+        pass
+
+
+class CommandResults:
+    """
+    This class should contain results of an integration command or a script
+    """
+    def __init__(self, outputs_prefix, outputs_key_field, outputs, indicators=None, readable_output=None,
+                 raw_response=None):
+        # type: (str, str, object, list, str, Any) -> None
+        self.indicators = indicators
+
+        self.outputs_prefix = outputs_prefix
+        self.outputs_key_field = outputs_key_field
+        self.outputs = outputs
+
+        self.raw_response = raw_response
+        self.readable_output = readable_output
+
+    def to_context(self):
+        outputs = {}  # type: dict
+        human_readable = None
+        raw_response = None
+
+        if self.indicators:
+            for indicator in self.indicators:
+                outputs.update(indicator.to_context())
+
+        if self.raw_response:
+            raw_response = self.raw_response
+
+        if self.outputs:
+            if not self.readable_output:
+                human_readable = tableToMarkdown('Results', self.outputs)
+            else:
+                human_readable = self.readable_output
+
+            if not self.raw_response:
+                raw_response = self.outputs
+
+            outputs_key = '{}(val.{} == obj.{})'.format(self.outputs_prefix, self.outputs_key_field, self.outputs_key_field)
+            outputs.update({
+                outputs_key: self.outputs
+            })
+
+        return_entry = {
+            'Type': EntryType.NOTE,
+            'ContentsFormat': EntryFormat.JSON,
+            'Contents': raw_response,
+            'HumanReadable': human_readable,
+            'EntryContext': outputs
+        }
+
+        return return_entry
+
+
+class DBotScore(Indicator):
+    NONE = 0
+    GOOD = 1
+    SUSPICIOUS = 2
+    BAD = 3
+
+    CONTEXT_PATH = 'DBotScore(val.Indicator && val.Indicator == obj.Indicator && val.Vendor == obj.Vendor)'
```

Comment:
i think that this path is ok only from 5.5

Reply:
It is good from 5.0, no?

Reply:
@hirshbergshachar on which version did you add the support of this? 
And @anara123 we should have two versions then, one for 5.0 and one for 5.5

Reply:
@ronykoz fix was merged only to 5.5 as far as I can see

Reply:
@anara123 So we should address this in the code.
Let's use the get_demisto_version function to check that we are >= 5.5 for this CONST
And otherwise, we should use the one without DT